### PR TITLE
Add checkboxes and radios conditional reveal examples

### PR DESCRIPTION
**Pull request by igloosi**
- [x] Guidance
- [x] Examples

Content written by @amyhupe

Trello Ticket
https://trello.com/c/jMwX1j2q/937-update-design-system-with-conditional-reveal

### CODE REVIEW

File: src/components/radios/conditional-reveal.njk [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      "text": "How do you want to be contacted?",
+      "isPageHeading": true
+    }
+  },
+  "items": [
+    {
+      "value": "email",
+      "text": "Email",
+      "conditional": {
+        "html": "<label class=\"govuk-label\" for=\"context-email\">Mobile phone number</label>\n<input class=\"govuk-input govuk-!-width-one-third\" name=\"context-email\" type=\"text\" id=\"context-email\">\n"
```
Comment (joelanman) [inline src/components/radios/conditional-reveal.njk]: Is it a clearer pattern to use `set` to define this html before the macro? eg

```
{% set emailHtml %}
  <label class="govuk-label" for="context-email">Mobile phone number</label>
  <input class="govuk-input govuk-!-width-one-third" name="context-email" type="text" id="context-email">
{% endset %}
```

Comment (igloosi) [reply src/components/radios/conditional-reveal.njk]: yes, hence the pr  has

Need to change to set, etc

Comment (joelanman) [reply src/components/radios/conditional-reveal.njk]: ha sorry! missed that

Comment (igloosi) [reply src/components/radios/conditional-reveal.njk]: np

**Pull request merged**